src/client/app/group-manager/group-manager.controller.js

```diff
@@ -26,6 +27,12 @@
     activate();
 
     function activate() {
+      var profile = ProfileAccess.getProfile();
+      if(profile) {
+        vm.authorized = profile.role_id == 1 || profile.role_id == 2;
+      } else {
+        $state.go('login');
```

Comment:
Shouldn't you just move them somewhere else if they aren't authorized? It seems to me the page would just break otherwise? Or even hide the option of going to this page in the nav bar if they aren't authorized